projects/lookup/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "@skyux/lookup",
+  "version": "5.0.0-beta.1",
+  "peerDependencies": {
+    "@angular/common": "^12.2.0",
+    "@angular/core": "^12.2.0",
+    "@angular/router": "^9.1.6 || ^12",
```

Comment:
This should be `^12.2.0`

Reply:
Hmmmm. I keep missing that. Not sure why it is ejecting like that. I’ll fix it. Thanks!